mesonbuild/interpreter/interpreter.py

```diff
@@ -2230,13 +2231,15 @@ def add_test(self, node: mparser.BaseNode,
         KwargInfo('subdir', (str, NoneType)),
         INSTALL_MODE_KW.evolve(since='0.47.0'),
         INSTALL_DIR_KW,
+        INSTALL_FOLLOW_SYMLINKS,
     )
     def func_install_headers(self, node: mparser.BaseNode,
                              args: T.Tuple[T.List['mesonlib.FileOrString']],
                              kwargs: 'kwtypes.FuncInstallHeaders') -> build.Headers:
         install_mode = self._warn_kwarg_install_mode_sticky(kwargs['install_mode'])
         source_files = self.source_strings_to_files(args[0])
         install_subdir = kwargs['subdir']
+        follow_symlinks = kwargs['follow_symlinks']
```

Comment:
I don't think it's necessary to make a temporary variable here, as we don't potentially change the value of follow_symlinks before passing it to `build.Headers`.

Reply:
I did those for brevity more than anything, but I can also just inline it.  Give me a moment.